src/web/app/pages-instructor/instructor-student-records-page/instructor-student-records-page.component.html

```diff
@@ -1,13 +1,20 @@
 <h1>{{studentProfile.name}}'s Records<small class="text-muted"> - {{courseId}}</small></h1>
 <tm-student-profile [studentProfile]="studentProfile" [photoUrl]="photoUrl"></tm-student-profile>
 <tm-more-info [studentName]="studentProfile.name" [moreInfoText]="studentProfile.moreInfo"></tm-more-info>
+<h2>Records in feedback sessions</h2>
 <div *ngFor="let session of sessions" class="card card-default mb-4">
   <div class="card-header font-weight-bold cursor-pointer" (click)="session.isCollapsed = !session.isCollapsed">
-    Feedback Session : {{session.name}}
+    Feedback Session : {{session.feedbackSession.feedbackSessionName}}
     <span class="float-right fas"
       [ngClass]="{'fa-chevron-up': !session.isCollapsed, 'fa-chevron-down': session.isCollapsed}"></span>
   </div>
   <div class="card-body" [ngbCollapse]="session.isCollapsed">
-    TODO
+    <tm-grq-rgq-view-responses *ngIf="session.questions.length > 0"
```

Comment:
One `responseGivenByStudents` will goes to here with `isGrq=true`.

Reply:
One `responseRecievedByStudents` will goes to here with `isGrq=flase`.

The rest will work magically.